### PR TITLE
Added solution for HW_2.3

### PR DESCRIPTION
**Pull request by polinakulyk**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -13,4 +16,16 @@
...
+
+        List<Machine> machines = new ArrayList<>();
+        machines.add(new Track());
+        machines.add(new Bulldozer());
+        machines.add(new Excavator());
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/MainApp.java]: first we should start all machines and then stop them. Read task closely)

Comment (polinakulyk) [reply src/main/java/core/basesyntax/MainApp.java]: thank you, I separated them in two loops (as it was before, and I clarified that at Q&A session)

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -13,4 +16,19 @@
  */
 public class MainApp {
 
+    public static void main(String[] args) {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/MainApp.java:20]: remove empty lines

Comment (polinakulyk) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**